xwalk_android.gypi

```diff
@@ -22,6 +22,7 @@
       ],
       'include_dirs': [
         '..',
+        '../third_party/mojo/src',
```

Comment:
I have updated the patch to include the mojo dependency. Although I'm not sure if mojo_environment_chromium is the right target.